modules/CustomView/models/Record.php

```diff
@@ -1306,6 +1313,34 @@ public static function getAllFilterByModule($module) {
 		return $instance;
 	}
 
+    public static function getDefaultFilterByModule($module)
```

Comment:
1. Missing type of $module. Is it a string, an object, or something else?
2. Missing phpDoc